src/main/java/seedu/address/model/UserPrefs.java

```diff
@@ -12,7 +12,7 @@
  * Represents User's preferences.
  */
 public class UserPrefs implements ReadOnlyUserPrefs {
-
+    //TODO: rename to "UserPrefs" once we confirm that we don't need AB3 files
```

Comment:
Should delete this since it's already renamed HAHAH
```suggestion
```

Reply:
Sorry, I missed it. 🌮 👨‍🦰